.circleci/config.yml

```diff
@@ -891,7 +891,7 @@ workflows:
   nightly-services-regression:
     triggers:
       - schedule:
-          cron: "0 21 * * *"
+          cron: "0 13,17,21 * * *"
```

Comment:
These hours are UTC times, you may want these regression jobs to run in our sleep times instead of the time when we are working. So maybe you want to change them to 4,8,12, which correspond to CST time 11PM, 3AM, 7AM. 

I am not sure your intention here, so you can ignore this comments if you have other concerns or restrictions.

Reply:
@ljianghedera  no preference for timings. I fixed based on your comment. Thank you!